File: C++/1561. Maximum Number of Coins You Can Get.cpp
class Solution {
public:
    int maxCoins(vector<int>& piles) {
        int n = piles.size();
        sort(piles.begin(),piles.end());
        int res=0;
        for(int i = n/3; i<n; i+=2){
            res += piles[i];
        }
        return res;
    }
};

File: JavaScript/1561. Maximum Number of Coins You Can Get.js
/**
 * @param {number[]} piles
 * @return {number}
 */
var maxCoins = function(piles) {
    var n = piles.length;
    piles.sort((a,b) => a-b);
    var res=0;
    for(var i = n/3; i<n; i+=2){
        res += piles[i];
    }
    return res;
};
